RELEASE.md

```diff
@@ -23,4 +23,11 @@ Keras pruning API:
 
  * Tested against TensorFlow 1.14.0, 2.0.0, and nightly.
 
-
+Keras clustering API:
+
+* New API for clustering of weights
+* Major Features
+  * Support for clustering of convolutional (except DepthwiseConv), fully connected and other commonly used standard Keras layers
```

Comment:
clustering of convolutional <- the "of" is not needed here

Reply:
Ok, I will change that.